packages/cli/config/jest.js

```diff
@@ -48,8 +48,9 @@ async function getConfig() {
 
     // Default behaviour is to not apply transforms for node_modules, but we still want
     // to apply the esm-transformer to .esm.js files, since that's what we use in backstage packages.
+    // The @kyma-project/asyncapi-react library needs to be transformed.
     transformIgnorePatterns: [
-      '/node_modules/(?!.*\\.(?:esm\\.js|bmp|gif|jpg|jpeg|png|frag|xml|svg)$)',
+      '/node_modules/(?!@kyma-project/asyncapi-react/)(?!.*\\.(?:esm\\.js|bmp|gif|jpg|jpeg|png|frag|xml|svg)$)',
```

Comment:
Yeah this is idd unfortunate. The page is using esm modules but really doesn't signal that in any way as far as I can tell. I don't wanna block this PR because of the issue so fine with override for now, but we do need to figure out a long-term solution. It would be possible to add this to `.jest.transformIgnorePatterns` inside `package.json` of the `api-docs` package, but that would have to be kept in sync with updates to this config so it isn't great either.

Reply:
@Rugvip I can also try to fix the original package so we don't get an issue here. But tbh, I am not really deep into jest and esm modules. Do you have a clue what might be wrong there? Would you say this issue is specific to the configuration in the backstage repository?

Reply:
It's a somewhat common override that's needed when using jest. The package is built in a way that makes it tricky to consume, and could definitely be left as feedback to the package author.

A common way to solve the issue is to build both CommonJS and ESModule outputs, and then point to each using `main` and `module` fields in `package.json`. It's puts more requirements on the build tooling though. For example, we do that here: https://github.com/spotify/backstage/blob/c4b6c1397577b89f117a2f663088315dda75aaa7/packages/catalog-model/package.json#L11